### PR TITLE
Adding pages

### PR DESCRIPTION
**Pull request by Will-lamuth**
- reworked html and css to match kanikas mockup
- added a research papers page
- added contact form on the Contact Us page

### CODE REVIEW

**Review by snowcov — APPROVED** [review]
Looks Good.

File: app.py [hunk 1]
```diff
@@ -224,5 +228,61 @@ def logout():
    # Redirect to login page
    return redirect(url_for('index'))
 
+@app.route('/aboutus', methods=['GET', 'POST'])
```
Comment (Pham-Vincent) [inline app.py:231]: Should Probably move this into its own File?

File: app.py [hunk 2]
```diff
@@ -224,5 +228,61 @@ def logout():
...
+def aboutus():
```
Comment (Pham-Vincent) [inline app.py:232]: Need to Comment The Use of this Function/ What this Function Does

File: app.py [hunk 3]
```diff
@@ -224,5 +228,61 @@ def logout():
...
+        cursor = conn.cursor(dictionary=True)
+        query = "SELECT * FROM Accounts where id = %s"
+        cursor.execute(query, (session['id'],))
+        account = cursor.fetchone()
+        cursor.close()
+        conn.close()
+        return render_template('aboutUs.html', account=account)
+    return render_template('aboutUs.html')
+
+def contactus(fname, lname, email, message):
```
Comment (Pham-Vincent) [inline app.py:255]: Need to Comment The Use of this Function/ What this Function Does

File: app.py [hunk 4]
```diff
@@ -224,5 +228,61 @@ def logout():
...
+    sender = "waterequitable@gmail.com"
+
+    #Change Receivers
+    receivers = ["waterequitable@gmail.com",]
+    #Change Receivers
```
Comment (Pham-Vincent) [inline app.py:260]: Extra Comment?

File: app.py [hunk 5]
```diff
@@ -224,5 +228,61 @@ def logout():
...
+        cursor.close()
+        conn.close()
+        return render_template('aboutUs.html', account=account)
+    return render_template('aboutUs.html')
+
+def contactus(fname, lname, email, message):
+    sender = "waterequitable@gmail.com"
+
+    #Change Receivers
+    receivers = ["waterequitable@gmail.com",]
```
Comment (Pham-Vincent) [inline app.py:259]: Are Sender And Receiver the Same? Can you comment what These are for exactly??

File: app.py [hunk 6]
```diff
@@ -224,5 +228,61 @@ def logout():
...
+    #Change Receivers
+
+    msg = MIMEText(message)
+    msg['Subject'] = email + ' , ' + fname + ' , ' + lname
+    msg['From'] = sender
+    msg['To'] = ', '.join(receivers)
+    with smtplib.SMTP_SSL('smtp.gmail.com', 465) as smtp_server:
+        smtp_server.login(sender, "rndxidrgmfzqkewq")
+        smtp_server.sendmail(sender, receivers, msg.as_string())
+    print("Message sent!")
```
Comment (Pham-Vincent) [inline app.py:269]: Can Delete This print Message

File: app.py [hunk 7]
```diff
@@ -224,5 +228,61 @@ def logout():
...
+
+    #Change Receivers
+    receivers = ["waterequitable@gmail.com",]
+    #Change Receivers
+
+    msg = MIMEText(message)
+    msg['Subject'] = email + ' , ' + fname + ' , ' + lname
+    msg['From'] = sender
+    msg['To'] = ', '.join(receivers)
+    with smtplib.SMTP_SSL('smtp.gmail.com', 465) as smtp_server:
```
Comment (Pham-Vincent) [inline app.py:266]: Explain What This is Really Quick?

File: app.py [hunk 8]
```diff
@@ -224,5 +228,61 @@ def logout():
...
+        smtp_server.login(sender, "rndxidrgmfzqkewq")
+        smtp_server.sendmail(sender, receivers, msg.as_string())
+    print("Message sent!")
+
+
+@app.route('/researchpapers')
+def research():
```
Comment (Pham-Vincent) [inline app.py:273]: Comment What This Function Does And Probably move it to its own file

File: static/json/ResearchPapers.json [hunk 9]
```diff
@@ -0,0 +1,58 @@
+[
```
Comment (Pham-Vincent) [inline static/json/ResearchPapers.json:1]: Probably Needs a Quick Comment at the Top for what this is used for

File: static/json/persons.json [hunk 10]
```diff
@@ -0,0 +1,142 @@
+[
```
Comment (Pham-Vincent) [inline static/json/persons.json:1]: Probably Needs a Quick Comment at the Top for what this is used for

**Review by Pham-Vincent — CHANGES_REQUESTED** [review]
Looks Good Just need to add some extra comments,

**Review by Pham-Vincent — APPROVED** [review]
Looks Good

**Pull request merged**